ethereum/Deploying.md

```diff
@@ -104,10 +152,11 @@ It will create a new file `PythUpgradable_merged.sol` which you can use in the e
 
 # Troubleshooting
 
+- You get `digital envelope routines::unsupported` error. The current truffle version is old and if you are using a new Node version you might need
```

Comment:
If you get `digital envelope routines::unsupported` error. it means that the current truffle version is old...